RELEASENOTES-1.4.docu

```diff
@@ -559,6 +559,20 @@
       implementations from unrelated templates to individually reference and
       call the implementations it overrides.</add-note></build-id>
   <build-id _6="6309" _7="7021"><add-note> Template <tt>function_io_memory</tt>
-      now searches for the bank in the same `subdevice`/`device` scope as the
-      `implement`. </add-note></build-id>
+      now searches for the bank in the same <tt>subdevice</tt>/<tt>device</tt> scope as the
+      <tt>implement</tt>. </add-note></build-id>
+  <build-id _6="next" _7="next"><add-note>Added a system
+      for <em>provisional language features</em>: language features
+      that are disabled by default but can be explicitly enabled with
+      a special syntax. </add-note></build-id>
```

Comment:
```
can be explicitly enabled per file via the new <tt>provisional</tt> statement,
which may only occur right after the DML version specification.
```

Reply:
At the very least, that it's per-file is important to note.

Reply:
Fixed.